judges/assignee-was-attached/assignee-was-attached.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+# SPDX-FileCopyrightText: Copyright (c) 2024-2025 Zerocracy
+# SPDX-License-Identifier: MIT
+
+# Judge that monitors open issue and create missing issue-was-assigned facts
+
+require 'fbe/conclude'
+require 'fbe/octo'
+
+Fbe.conclude do
+  quota_aware
+  on "(and (eq where 'github') (eq what 'issue-was-opened'))"
+
+  consider do |f|
+    next if Fbe.fb.query(
+      "(and
+         (eq where 'github')
+         (eq repository #{f.repository})
+         (eq issue #{f.issue})
+         (eq what 'issue-was-closed'))"
+    ).each.to_a.first
+    Fbe.octo.issue_events(
```

Comment:
@Yegorov it seems, we will go to GitHub API for every `issue-was-opened` fact, even if it already has `issue-was-assigned`. Maybe you've made a typo above?

Reply:
@yegor256 yes of course, because you can assign multiple users or reassign to another user

Reply:
@yegor256 Can issue be assigned to multiple users? Or just only one? And not been reassigned?

Reply:
@Yegorov indeed, it can be assigned many times. We don't care about all events. We are only interested in who it's assigned to, at this very moment.

Reply:
@yegor256 ok, thanks, fixed.